src/System.Security.Cryptography.Primitives/tests/CryptoStream.netcoreapp.cs

```diff
@@ -64,6 +64,31 @@ public static void DisposeAsync_DerivedStream_InvokesDispose()
             }
         }
 
+        [Theory]
+        [InlineData("Aes")]
+        [InlineData("Rijndael")]
+        public static void PaddedAesRijdnael_PartialRead_Success(string algName)
+        {
+            SymmetricAlgorithm alg = SymmetricAlgorithm.Create(algName);
```

Comment:
Since the test is about CryptoStream and not the input, can we just scope it down to Aes.Create() and eliminate the theory (and, what I'm really after, eliminate the call to SymmetricAlgorithm.Create, and also the use of Rijndael? :smile:)

Reply:
Done